src/cli/main.py

```diff
@@ -131,9 +132,10 @@ def create_cmd(
     \b
     Make a transaction to create a clawback coin
     """
-    amount = int(float(amount) * MOJO_CONST)
-    fee = int(float(fee) * MOJO_CONST)
-    async def do_command(fingerprint):
+    final_amount = int(Decimal(amount) * MOJO_CONST)
```

Comment:
I might go the other way here.  Rename `amount` to `amount_str` or `raw_amount`.  You can assign an arbitrary parameter name to an option by passing that string after the flag strings.  `"-m", "--fee", "raw_fee",`.  Someday we'll just make Click types for these things so it's all embedded with the checks and conversions and so on.

Reply:
Thanks, fixed